packages/react-rest/README.md

```diff
@@ -20,18 +20,18 @@
     - [Creating a todo with a user](#creating-a-todo-with-a-user)
 - [Alternatives to hooks](#alternatives-to-hooks)
 
-Unlike code generation tools which allow you to write your schema and then generate your code, once you start making customizations you can't re-run the generator without losing your customization HatchifyJS enables you to make changes to your database schema and customize app behavior **independently**. This is because HatchifyJS is **not** code generation—it's a system of modular and hierarchical libraries that can be consumed piecemeal to use as much or as little of HatchifyJS abilities as you require.
-
```

Comment:
This paragraph appears to have been inadvertently copy/pasted? It's from the root README and doesn't appear to be particularly relevant to this guide